.github/workflows/multi-arch-test-build.yml

```diff
@@ -57,6 +57,15 @@ jobs:
             runtime_test: true
 
     steps:
+      - name: Free up some diskspace
```

Comment:
Would it be better to use something like. ``Remove unused: Android SDK, .NET, Swift"?